### ISSUE TITLE
Usage with other game jams

### ISSUE THREAD

**Issue by d-bucur** [GameMakersToolkit/team-finder#286]
Sorry, not really an issue, but this app is amazing! Any chance this could be opened up for use in different game jams?

**Comment by Willdotwhite** [comment]
Hey @d-bucur , thanks for the feedback!

The app is available for any other jam organisers to use - check the (somewhat out of date) Wiki! - and we'll hopefully be making it easier to use for other communities going forwards, once we tidy things up a bit :D

**Comment by d-bucur** [comment]
Oh yeah, self hosting it definitely. But I doubt many organizers would go through that unless it's a huge jam. What I meant was more like on the same instance that it is running now, possibly adding a field for the game jam for each entry.

**Comment by Willdotwhite** [comment]
Ahh, like a tool which manages multiple jams?

We've talked about that a few times - I'd absolutely love to set something
like that up (maybe in a WordPress-style subdomain model), but if we build
a managed service then we need to consider content moderation, which is
less fun!

Unless I've misunderstood what you meant?

**Comment by d-bucur** [comment]
Yes, I meant using the same tool for multiple jams. You wouldn't really need to deploy it to a different sub domain, I think it could work from a single deployment. The way I see it is you would select the specific jam when you post an entry and then people could either filter for a specific jam, or they could look for pair ups in potential jams that they hadn't considered.

The problem here is that you would need a reliable way to populate the selectable jams, or maybe have a free field where the user pastes in the url.

But yes you're right that it would require more content moderation

**Comment by Willdotwhite** [comment]
Hi again @d-bucur ! Long time no message, but you might be interested to hear this is something we're exploring!

There's a branch in development now (`feature/multiple-jams`) which provides support for this - I decided to try my luck and hope the moderation hassle doesn't become too bad 😂 

There won't be any movement on this before the next GMTK jam; I've got an 11 week old baby now and given that I can't concretely commit time to bugfixing, I don't want to make a massive change to the codebase that I might not be able to support!
